src/main/groovy/uk/gov/hmrc/jenkinsjobbuilders/domain/scm/GitHubScm.groovy

```diff
@@ -32,7 +32,11 @@ final class GitHubScm implements Scm {
         gitHubScm(host, repository, branch, protocol, null, null, null)
     }
 
-    static GitHubScm gitHubScm(String host, String repository, String branch, String protocol, String refspec, String credentials, String name = null, int depth = 0, boolean honorRefspec = false, boolean pullTags = true, int timeout = 10) {
+    static GitHubScm gitHubScm(String host, String repository, String branch, String protocol, String refspec, String credentials, String name = null, int depth = 0, boolean honorRefspec = false, boolean pullTags = true) {
+        new GitHubScm(host, repository, branch, protocol, refspec, credentials, name, depth, honorRefspec, pullTags)
```

Comment:
instead of defaulting on the constructor args, pass the timeout parameter to the constructor here.. so add ", 0" on the end